<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insere_form</class>
 <widget class="QDialog" name="insere_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>insere_form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>381</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nome Do produto</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nomeProduto"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Quantidade Mínima</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="quantidadeMinimaProduto"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Tamanho</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="nivelPrioridade"/>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="botaoCadastra">
      <property name="text">
       <string>Cadastrar</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="retorno">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Preço</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="spinBoxPreco"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
